### PR TITLE
Register listeners to Nettrine\Dbal EventManager

### PR DESCRIPTION
**Pull request by PetrHudik**

### CODE REVIEW

File: src/DI/DoctrineExtensionsExtension.php [hunk 1]
```diff
@@ -13,10 +13,13 @@
 use Gedmo\Translatable\TranslatableListener;
 use Gedmo\Tree\TreeListener;
 use Nette\DI\CompilerExtension;
+use Nettrine\DBAL\DI\DbalExtension;
```
Comment (f3l1x) [inline src/DI/DoctrineExtensionsExtension.php]: This is not in composer deps, is it?

Comment (PetrHudik) [reply src/DI/DoctrineExtensionsExtension.php]: No, it's not. :cry: You're right!

I'm using it for const [few lines lower](https://github.com/nettrine/extensions/pull/7/commits/dc831ec7c6db0b6861acc65265a78faea58988b7#diff-6408d50bd89dfaa07bbafbe4fe997be6R21). Souhld I just change const to something like this:

`public cont TAG_NETTRINE_SUBSCRIBER = 'nettrine.subscriber'`?

Comment (f3l1x) [reply src/DI/DoctrineExtensionsExtension.php]: Sure.

**Review by f3l1x — CHANGES_REQUESTED** [review]

File: .docs/README.md [hunk 2]
```diff
@@ -50,6 +50,16 @@ orm.annotations:
         - Gedmo\Tree\Entity
 ```
 
+If you using Nettrine\Dbal all listeners are registered automatically, or you have to register it manually:
```
Comment (f3l1x) [inline .docs/README.md]: If you are using `nettrine/dbal` ...

**Review by PetrHudik — COMMENTED** [review]
It's done. 

(I still see, that changes are requested, but I can't change it, because I allready commit and push them.)

**Comment by PetrHudik** [comment]
@f3l1x  is it ok?

**Comment by f3l1x** [comment]
Thanks Petr ;-)

**Pull request merged**